app/controllers/companies_controller.rb

```diff
@@ -0,0 +1,45 @@
+class CompaniesController < ApplicationController
```

Comment:
Check entire class. Indentation is is not done properly

Refer https://github.com/bbatsov/ruby-style-guide#spaces-indentation